CI-DNS.yml

```diff
@@ -0,0 +1,144 @@
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys:
+        - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC1JnaVPFjGDuhzVUTNSug8TPqyCO14rsxSJ6fJiWJTj9LkqSoJ7SJy0NiLPWXygsnM6cvibpPZrDrmwqGHwTLlhnurbEUA8EvsddxOPkOzX+diMd6xthFMcdTYXS+sQQ9cPB6otrZai7GsAiT7OEQutF7g1JXIpzt1jBd8P+S3p5B5Av+AuDkSHwwvC8Y0IREQLBzitCQP9I9Mrw5KVS5uwVHajYri0s8ZvsXXJtxBUaWjRxFpyjvkL0NDPanfwdxQP3X0z2crY/RMr72dwjk0P74QQmfoD9juD3J8/9BkdWKLuhQTvTShduqhAJK9XXj2/pQj/IuM1ZSALVIW/Dgr
```

Comment:
Should the key be hardcoded? Perhaps it could be injected like we do [in run_ocp.sh](https://github.com/shiftstack-dev-tools/shiftstack-ci/blob/b99992c027f77914bcf9653448957351ede00381/run_ocp.sh#L77-L78).

Reply:
I think we actually do want a hardcoded key here. However, it should be a shared key for the team and not my personal key.

The reason to hardcode it, is that ideally we don't edit the configuration outside of code review. For example, when we want/need to recreate the CI-DNS VM, we should do it exactly from the code checked into this repo. That way we maintain this repo as the source of truth.

Reply:
OK, it does indeed make sense to hardcode the key in that case. There are a few ways to share secrets via git such as https://git-secret.io/ is one of them. It's even packaged for the major distros.